Makefile

```diff
@@ -110,8 +110,11 @@ clean:
 doc-setup:
 	pip install -r docs/requirements.txt
 doc: $(MANIFEST)
+	mkdir -p ./docs/plugins/ ./build/plugin_docs
+	antsibull-docs collection --use-current --squash-hierarchy --dest-dir ./build/plugin_docs theforeman.foreman
+	cp -a ./build/plugin_docs/* ./docs/plugins/
```

Comment:
@mdellweg is that OK with you? It generates the docs in `build`, so we don't need to clean them up, but we need them in `docs/plugins` for Sphinx to pick them up later.

Reply:
Maybe we should add `docs/plugins` to the `clean` target.

For me to understand better: Docs are only rebuilt, if MANIFEST changed, which means something actually changed. I think, that's OK.

Reply:
Right, and then we can just build into that folder instead of copying things around.

Reply:
I guess